backend/btrixcloud/pages.py

```diff
@@ -560,6 +569,84 @@ async def list_pages(
 
         return [PageOut.from_dict(data) for data in items], total
 
+    async def list_collection_pages(
+        self,
+        coll_id: UUID,
+        org: Optional[Organization] = None,
+        url: Optional[str] = None,
+        url_prefix: Optional[str] = None,
+        ts: Optional[datetime] = None,
+        page_size: int = DEFAULT_PAGE_SIZE,
```

Comment:
Should we also add an option to filter by depth (once its added from the other PR)?

Reply:
Note to self, we should also add depth other new page fields as sort options